client/components/button/README.md

```diff
@@ -19,6 +19,7 @@ render: function() {
 
 * `compact`: (bool) whether the button is compact or not.
 * `primary`: (bool) whether the button is styled as a primary button.
+* `secondary`: (bool) whether the button is styled as a secondary button.
```

Comment:
Button is a very widely used component, and AFAIK its semantics and style strictly reflect Calypso design. The way I see it, any default (i.e. not `primary`, nor `borderless`) button in Calypso constitutes a "secondary" button of sorts. In our Guides step, our special gray button is, semantically, the same as those default buttons.

To put it much more plainly: it feels like our button is the same as the default button, and we actually just want _specific CSS_ for `<Button />` if it's inside a Guides step.